specification/appconfiguration/resource-manager/Microsoft.AppConfiguration/stable/2023-03-01/appconfiguration.json

```diff
@@ -821,7 +821,7 @@
         "tags": [
           "KeyValues"
         ],
-        "description": "Gets the properties of the specified key-value.",
+        "description": "Gets the properties of the specified key-value. NOTE: This operation is intended for use in ARM Template deployments. The data plane API should be used for other interactions with App Configuration key-values.",
```

Comment:
Not sure if people will interpret "other interactions" as "other operations but this one". I know you mean "other scenarios but the ARM template".

Do you feel if it makes sense to include a link to our data-plane REST API doc?
https://go.microsoft.com/fwlink/?linkid=2078296

Reply:
> Do you feel if it makes sense to include a link to our data-plane REST API doc?

I was a bit on the fence about it. Wasn't sure if it made sense to link to the dataplane REST API spec (in this repo), the link you linked, or SDK link which ultimately this is used to build.

I'm fine to add the link if you'd like.

Reply:
> NOTE: This operation is intended for use in ARM Template deployments. For all other scenarios involving App Configuration key-values the data plane API should be used instead.

Love it.

> I was a bit on the fence about it. 

I was just throwing an idea. If a link looks like an outlier to you, I'm okay not having it.

Reply:
I'll omit the link then.

Updated the wording, thanks !